tests/test_bom_handler.py

```diff
@@ -80,33 +86,54 @@ def _compare_boms(*, source_bom: str, result_bom: str):
             output_lines.append(diff_item)
         return output_lines
 
+    def test_roundtrip_from_text_with_assertions(self):
```

Comment:
The diff here is garbled. I basically duplicated the two existing tests, but passing in a path instead of a string